API.md

```diff
@@ -640,7 +640,9 @@ const schema = Joi.any().strict();
 Sets a default value if the original value is undefined where:
 - `value` - the value.
   - `value` supports [references](#refkey-options).
-  - `value` may also be a function which returns the default value. If `value` is specified as a function that accepts a single parameter, that parameter will be a context object that can be used to derive the resulting value. **This clones the object however, which incurs some overhead so if you don't need access to the context define your method so that it does not accept any parameters**.
+  - `value` may also be a function which returns the default value. If `value` is specified as a function that accepts a single parameter, that parameter will be a context object that can be used to derive the resulting value.
```

Comment:
This change is slightly misleading. The use of function with arguments is what creates clones, not the simple fact to use a function, which is why I kept it like that.

Reply:
I inserted a line about the default value for dates and the use of a function. Other than cleaning up the grammar slightly on the intro to the comment, it was just moved to a sub bullet. I'm not sure how that changes the intent of the original.

The language change was `**This clones the object however` to `**Caution: this clones the object**,`

Reply:
The single block made it clear in my head, maybe only in my head :)
Repeating in the bullet point might help with that.